doc/classes/OmniLight.xml

```diff
@@ -5,7 +5,7 @@
 	</brief_description>
 	<description>
 		An Omnidirectional light is a type of [Light] that emits light in all directions. The light is attenuated by distance and this attenuation can be configured by changing its energy, radius, and attenuation parameters.
-		[b]Note:[/b] Due to current rendering engine limitations, only 8 OmniLights may affect a single mesh [i]resource[/i] at once. Consider splitting your level into several meshes to decrease the likelihood that more than 8 lights will affect the same mesh resource. Splitting the level mesh will also improve frustum culling effectiveness, leading to greater performance.
+		[b]Note:[/b] By default, only 32 OmniLights may affect a single mesh [i]resource[/i] at once. Consider splitting your level into several meshes to decrease the likelihood that more than 32 lights will affect the same mesh resource. Splitting the level mesh will also improve frustum culling effectiveness, leading to greater performance. If you need to use more lights per mesh, you can increase [member ProjectSettings.rendering/limits/rendering/max_lights_per_object] at the cost of shader compilation times.
```

Comment:
I don't know if the comment / advice on mesh splitting is still relevant though since you can bump up the max number.
Maybe this should just be a note referencing the project settings (without saying its value which could change and would thus need to be updated here).

Reply:
Using a lot of lights on a large meshes will still be much slower, so I'd recommend bumping the limit only as a last resort.

Reply:
Alright :)